v5/pivotal/projects.go

```diff
@@ -116,3 +116,23 @@ func (service *ProjectService) Get(projectId int) (*Project, *http.Response, err
 
 	return &project, resp, err
 }
+
+func (service *ProjectService) AddStory(
```

Comment:
@dav Not sure what you did, but it seems like this view is somehow badly generated. Perhaps you should clean up your commits not to send merge commits etc. These should not be in the pull request...